packages/docs/docs/concepts/cornerstone-tools/tools.md

```diff
@@ -158,3 +158,44 @@ interactions.
     </td>
   </tr>
 </table>
+
+## Tool interpolation
+
+Some of tools can be configured to use interpolation process. PlanarFreehandROITool is an example for that.
```

Comment:
I don't think it is "some of tools" - interpolation is really specific to the freehand tool, or anything else creating a region like the freehand tool.  Maybe something like "Contour tools like the FreehandTool can be configured..."